.github/workflows/build-package-docs.yaml

```diff
@@ -0,0 +1,99 @@
+name: Ansible package docs build
+on:
+  workflow_dispatch:
+    inputs:
+      github_fork:
+        description: 'GitHub Fork'
+        required: true
+        default: 'ansible'
+      github_branch:
+        description: 'GitHub Branch'
+        required: true
+        default: 'devel'
+      language:
+        type: choice
+        description: 'Language'
+        required: true
+        default: 'english'
+        options:
+        - 'english'
+        - 'japanese'
+      package_version:
+        type: choice
+        description: 'Ansible Version'
+        required: true
+        default: 'devel'
+        options:
+        - 'devel'
+        - '9'
+        - '8'
+        - '7'
+        - '6'
+        - '5'
+        - '4'
+        - '3'
+        - '2.10'
+      latest_symlink:
```

Comment:
I don't think we can publish most of these releases at this point, in the jenkins job. I suggest we just start with Ansible 9 and later, or if we want a few more, only the ones with nox support (down to Ansible 7 but we should test to see if it works there... some backports fail on core 2.14 so it may not).